content/en/real_user_monitoring/mobile_and_tv_monitoring/advanced_configuration/ios.md

```diff
@@ -268,99 +268,99 @@ Datadog.setUserInfo(id: "1234", name: "John Doe", email: "john@doe.com")
+`trackFrustrations`
+: Determines whether automatic tracking of user frustrations is enabled. By default, this is set to `true`.
 
-`actionEventMapper`
-: Sets the data scrubbing callback for actions. This can be used to modify or drop action events before they are sent to Datadog. For more information, see [Modify or drop RUM events](#modify-or-drop-rum-events).
+`trackBackgroundEvents`
+: Determines whether RUM events are tracked when no view is active. By default, this is set to `false`.
 
-`errorEventMapper`
-: Sets the data scrubbing callback for errors. This can be used to modify or drop error events before they are sent to Datadog. For more information, see [Modify or drop RUM events](#modify-or-drop-rum-events).
+`trackWatchdogTerminations`
+: This setting determines whether the SDK should track application terminations performed by Watchdog. The default setting is `false`.
 
-`longTaskEventMapper`
-: Sets the data scrubbing callback for long tasks. This can be used to modify or drop long task events before they are sent to Datadog. For more information, see [Modify or drop RUM events](#modify-or-drop-rum-events).
+`uiKitActionsPredicate`
+: Enables tracking user interactions (taps) as RUM actions. You can use the default implementation of `predicate` by setting the `DefaultUIKitRUMActionsPredicate` or implement [your own `UIKitRUMActionsPredicate`](#automatically-track-user-actions) customized for your app.
 
-`onSessionStart`
-: Enter a method that gets called when RUM starts the session.
+`uiKitViewsPredicate`
+: Enables tracking `UIViewControllers` as RUM views. You can use default implementation of `predicate` by setting the `DefaultUIKitRUMViewsPredicate` or implement [your own `UIKitRUMViewsPredicate`](#automatically-track-views) customized for your app.
 
-`customEndpoint`
-: A custom server URL for sending RUM data.
+`urlSessionTracking`
+: Enables tracking `URLSession` tasks (network requests) as RUM resources. The `firstPartyHostsTracing` parameter defines hosts that are categorized as `first-party` resources (if RUM is enabled) and have tracing information injected (if tracing feature is enabled). The `resourceAttributesProvider` parameter defines a closure to provide custom attributes for intercepted resources that is called for each resource collected by the RUM iOS SDK. This closure is called with task information and may return custom resource attributes or `nil` if no attributes should be attached.
 
-`telemetrySampleRate`
-: The sampling rate for the SDK internal telemetry utilized by Datadog. This must be a value between `0` and `100`. By default, this is set to `20`.
+`viewEventMapper`
+: Sets the data scrubbing callback for views. This can be used to modify view events before they are sent to Datadog. For more information, see [Modify or drop RUM events](#modify-or-drop-rum-events).
```

Comment:
```suggestion
: The data scrubbing callback for views. This can be used to modify view events before they are sent to Datadog. For more information, see [Modify or drop RUM events](#modify-or-drop-rum-events).
```